### PR TITLE
UI Wording & Raw Values

### PR DESCRIPTION
**Pull request by paulirish**
Three small things:

1. Update some wording in the UI
2. Show raw values in the extension UI (theme-color and FMP)
3. disable sourcemaps for extension since they don't work and just makes debugging harder so far.

### CODE REVIEW

File: audits/manifest/short-name-length.js [hunk 1]
```diff
@@ -56,7 +56,8 @@ class ManifestShortNameLength extends Audit {
       // See #69 for more discussion.
       isShortNameShortEnough = (manifest.short_name.value.length <= suggestedLength);
       if (!isShortNameShortEnough) {
-        debugString = `${suggestedLength} chars is the suggested maximum length`;
+        debugString = `${suggestedLength} chars is the suggested maximum homescreen label length.`;
```
Comment (brendankenny) [inline audits/manifest/short-name-length.js]: nit: move the period after the following parenthetical

File: audits/performance/first-meaningful-paint.js [hunk 2]
```diff
@@ -32,14 +32,14 @@ class FirstMeaningfulPaint extends Audit {
    * @override
    */
   static get name() {
-    return 'first-meaningful-paint';
+    return 'first-contentful-paint';
```
Comment (brendankenny) [inline audits/performance/first-meaningful-paint.js]: :+1:

File: extension/app/scripts.babel/background.js [hunk 3]
```diff
@@ -64,11 +64,11 @@ function createResultsHTML(results) {
     item.score.subItems.forEach(subitem => {
       const debugString = subitem.debugString ? ` title="${escapeHTML(subitem.debugString)}"` : '';
 
-      // TODO: make this work with numeric values.
       const status = subitem.value ?
           `<span class="pass" ${debugString}>Pass</span>` :
           `<span class="fail" ${debugString}>Fail</span>`;
-      groupHTML += `<li>${escapeHTML(subitem.description)}: ${status}</li>`;
+      const rawValue = subitem.rawValue ? `(${subitem.rawValue})` : '';
+      groupHTML += `<li>${escapeHTML(subitem.description)}: ${status} ${rawValue}</li>`;
```
Comment (brendankenny) [inline extension/app/scripts.babel/background.js]: need to `escapeHTML` the `rawValue`

Comment (paulirish) [inline extension/app/scripts.babel/background.js]: thx for that catch​

File: extension/gulpfile.babel.js [hunk 4]
```diff
@@ -89,17 +89,17 @@ gulp.task('babel', () => {
     'app/scripts.babel/app.js',
     'app/scripts.babel/chromereload.js',
     'app/scripts.babel/background.js'])
-    .pipe($.sourcemaps.init())
+    // .pipe($.sourcemaps.init())
```
Comment (brendankenny) [inline extension/gulpfile.babel.js]: just delete? Our transformation isn't complex, and in theory we don't want any more than the module concat anyways, so I don't think we'll want it back

Comment (brendankenny) [inline extension/gulpfile.babel.js]: can probably uninstall `gulp-sourcemaps` as well

**Comment by brendankenny** [comment]
> disable sourcemaps for extension since they don't work and just makes debugging harder so far.

:tada: they're pretty annoying :)

**Comment by brendankenny** [comment]
1 real thing and some nits. LGTM

File: extension/gulpfile.babel.js (hunk 4 above)
Comment (paullewis) [inline extension/gulpfile.babel.js]: Yeah AFAIK we only want browserify anyway, because Chrome 52 already knows all the ES2015 stuffs.

Comment (paulirish) [inline extension/gulpfile.babel.js]: sg. removed sourcemaps entirely.

**Comment by paulirish** [comment]
all done.

**Comment by paullewis** [comment]
Just waiting on tests, then I'll merge.

File: extension/app/scripts.babel/background.js [hunk 5]
```diff
@@ -64,11 +64,11 @@ function createResultsHTML(results) {
     item.score.subItems.forEach(subitem => {
       const debugString = subitem.debugString ? ` title="${escapeHTML(subitem.debugString)}"` : '';
 
-      // TODO: make this work with numeric values.
       const status = subitem.value ?
           `<span class="pass" ${debugString}>Pass</span>` :
           `<span class="fail" ${debugString}>Fail</span>`;
-      groupHTML += `<li>${escapeHTML(subitem.description)}: ${status}</li>`;
+      const rawValue = subitem.rawValue ? `(${escapeHTML(subitem.rawValue)})` : '';
```
Comment (brendankenny) [inline extension/app/scripts.babel/background.js]: does this do weird things with non-string `subitem.rawValue`s?

Comment (brendankenny) [inline extension/app/scripts.babel/background.js]: yeah, if this is a number, for instance, that's no good. You could put the call to `escapeHTML` around the template string instead

Comment (paullewis) [inline extension/app/scripts.babel/background.js]: Sigh, didn't see this. Guess we'll need a PR to fix.

**Pull request merged**